designsafe/apps/workspace/admin.py

```diff
@@ -48,10 +61,10 @@ def get_fieldsets(self, request, obj=None):
                 },
             ),
             (
-                "HTML App Body for app_type: HTML",
+                "HTML or External app information",
                 {
                     "classes": ["collapse"],
-                    "fields": ["html"],
+                    "fields": ["download_href", "html",],
```

Comment:
@heddabedda and I chatted. External makes sense, because of other possible use cases:

- list of downloads (our actual use case)
- agreement page before download
- direct download
- instructions for download